AdminServer/appscale/admin/utils.py

```diff
@@ -23,7 +20,11 @@
   JAVA,
   SOURCES_DIRECTORY,
   Types,
-  UNPACK_ROOT
+  UNPACK_ROOT,
+  REQUIRED_PULL_QUEUE_FIELDS,
+  REQUIRED_PUSH_QUEUE_FIELDS,
+  SUPPORTED_PULL_QUEUE_FIELDS,
+  SUPPORTED_PUSH_QUEUE_FIELDS
 )
```

Comment:
It's better to use a single import style at least in scope of a single module. So it looks reasonable to import all needed from constants as it's done here (`from .constants import (...)`) and not use `from . import constants`.
Another benefit of doing so will be using one-line raise exception statements.